src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -7,6 +7,7 @@
 using System;
 using System.Collections.Immutable;
 using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
```

Comment:
Is this using needed?

Reply:
It won't be after the refactoring you suggested, but it was for this commit, as I used `NotNullIfNotNull`